### DISCUSSION TITLE
Could you add a Dark/Light Theme switch on the page ?

### DISCUSSION THREAD

**Discussion by slashformotion** [jpanther/congo#15] [Ideas]
It would be awesome, it could be activated via a boolean in the settings

**Comment by jpanther** [comment]
Thanks for the suggestion. This is a feature that is on my roadmap and I hope to get it into a future release. It would definitely be something that could be toggled on and off. I was also thinking of adding a config parameter for forcing light or dark mode too.

**Reply by jpanther** [reply]
I've made some changes on the dev branch that adds some new theme parameters for this. You can now set `darkToggle = true` and a button will be added to the page footer to switch the appearance. You can also set a new `darkMode` parameter to force either dark (`true`) or light (`false`) appearance.

**Reply by jpanther** [reply]
This is now available in v1.5.0
